js/parts/Stacking.js

```diff
@@ -284,12 +284,20 @@ Chart.prototype.getStacks = function () {
  * @return {void}
  */
 Axis.prototype.buildStacks = function () {
-    var axisSeries = this.series, reversedStacks = pick(this.options.reversedStacks, true), len = axisSeries.length, i;
+    var axisSeries = this.series, reversedStacks = pick(this.options.reversedStacks, true), len = axisSeries.length, actualSeries, options, i;
     if (!this.isXAxis) {
         this.usePercentage = false;
         i = len;
         while (i--) {
-            axisSeries[reversedStacks ? i : len - i - 1].setStackedPoints();
+            actualSeries = axisSeries[reversedStacks ? i : len - i - 1];
+            options = actualSeries.options;
```

Comment:
I guess that `options` are series options. If yes, I think that `seriesOptions` will be more intuitive.

Reply:
I agree, corrected. Thanks!